### ISSUE TITLE
esc button behaviour in game shelf

### ISSUE THREAD

**Issue by Ingegneus** [ArnoldSmith86/virtualtabletop#1555]
I find myself instinctively pressing the esc button whenever there is some sort of overlay in the game shelf, but I end up closing the shelf entirely. what is the sentiment of others towards this behaviour?

**Comment by RaphaelAlvez** [comment]
I wouldn't mind changing the behavior to move back one overlay.

**Comment by ArnoldSmith86** [comment]
Yeah, doesn't sound like a bad idea. In general, ESC should probably just press an `X` button whenever there is one.
